src/rules/lowercase-name.ts

```diff
@@ -52,6 +53,32 @@ const testDescription = (argument: ArgumentLiteral): string | null => {
   return argument.quasis[0].value.raw;
 };
 
+const jestTopFunctioName = (
+  node: CallExpressionWithCorrectCalleeAndArguments,
+  counter: number,
+) => {
+  const description = testDescription(node.arguments[0]);
+  if (description === null) {
+    return null;
+  }
+
+  if (counter !== 1) {
```

Comment:
You could do this check first, before anything else to reduce the work being done slightly; alternatively this could be merged w/ the `description === null` check.